### ISSUE TITLE
markus submission, no submit button?

### ISSUE THREAD

**Issue by fares-soliman** [karansher/computer-graphics-raster-images#85]
When you upload files onto markus, are the uploaded files your submission, or do you have to submit it somehow? Don't see a submit button or anything...

**Comment by abhimadan** [comment]
Yes, your uploaded files are your submission. Their interface is confusing but once you've selected files to upload, and you see them on MarkUs, you've submitted.
